notebooks/post-startup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Default post startup script for AI notebooks.
+# AI Notebook post startup scrips are run only when the instance is first created.
+
+# Send stdout and stderr from this script to a tmp file for debugging.
+exec >> /tmp/post-startup-output.txt
+exec 2>&1
+set -x
+
+# The linux user that JupyterLab will be running as. It's important to do some parts of setup in the
+# user space.
+JUPYTER_USER="jupyter"
```

Comment:
If a variable is not expected to change, then recommend using "readonly":

```
readonly JUPYTER_USER="jupyter"
```

See https://google.github.io/styleguide/shellguide.html#quoting